frontend/src/app/features/assistance-selection/assistance-selection.component.ts

```diff
@@ -0,0 +1,24 @@
+import { Component } from '@angular/core';
+import config from "../../core/constants/config.json"
+import { AssistanceLevel } from '../../core/interfaces/assistance-level';
+
+@Component({
+  selector: 'assistance-selection',
+  templateUrl: './assistance-selection.component.html',
+  styleUrls: ['./assistance-selection.component.scss']
+})
+export class AssistanceSelectionComponent {
+  levels: AssistanceLevel[] = config['assistance-levels']
+  selectedLevel: string = ''
```

Comment:
This variable should be just `selected` to keep the language the same between components.